src/main/scala/by/itechart/tutorial/web/GroupsApiRouter.scala

```diff
@@ -1,53 +1,75 @@
 package by.itechart.tutorial.web
 
-import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
-import akka.http.scaladsl.model.StatusCodes
 import akka.http.scaladsl.server.Directives.{complete, concat, get, onSuccess, pathPrefix, _}
 import akka.http.scaladsl.server.Route
-import by.itechart.tutorial.dao.{Group, GroupRepository}
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import by.itechart.tutorial.dao.Group
 import by.itechart.tutorial.service.GroupService
-import by.itechart.tutorial.util.const.Constants.MODEL_NOT_FOUND
+import by.itechart.tutorial.util.const.Constants.{DELETE_GROUP_EXCEPTION_MESSAGE, GROUP_NOT_FOUND, POST_GROUP_EXCEPTION_MESSAGE, PUT_GROUP_EXCEPTION_MESSAGE}
 import javax.inject.Inject
+import org.json4s.native.Serialization.{read, write}
+import org.json4s.{DefaultFormats, Formats}
+
+import scala.concurrent.duration._
+import scala.util.Success
 
 class GroupsApiRouter @Inject()(groupService: GroupService) {
 
   def getGroupsApiRoutes: Route = route
 
-  import by.itechart.tutorial.util.marshalling.GroupJsonProtocol._
+  implicit val formats: Formats = DefaultFormats
+
+  import by.itechart.tutorial.Application._
 
   private val route: Route =
     concat(
       get {
         pathPrefix("groups" / LongNumber) { id =>
           val maybeItem = groupService.getGroupById(id)
           onSuccess(maybeItem) {
-            case Some(group) => complete(group)
-            case None => complete(MODEL_NOT_FOUND.format("group"))
+            case Some(group) => complete(write(group))
```

Comment:
a guess write could be an implicit function, so where will be no need to call it directly